enginetest/queries/queries.go

```diff
@@ -9059,11 +9119,11 @@ var ErrorQueries = []QueryErrorTest{
 	},
 	{
 		Query:          `select JSON_EXTRACT('{"id":"abc"}', '$.id')-1;`,
-		ExpectedErrStr: `error: 'abc' is not a valid value for 'double'`,
+		ExpectedErrStr: `number has no digits`,
```

Comment:
Interesting! This one and the following query both do execute successfully in MySQL: 

```sql
mysql> select JSON_EXTRACT('{"id":"abc"}', '$.id')-1;
+----------------------------------------+
| JSON_EXTRACT('{"id":"abc"}', '$.id')-1 |
+----------------------------------------+
|                                     -1 |
+----------------------------------------+
1 row in set, 1 warning (0.01 sec)
```

Seems like it's probably a bug with the `JSON_EXTRACT` function though, and not your change. Could you take a closer look to confirm that and open a tracking issue for this new JSON_EXTRACT bug if it's a separate issue, so that we've got it in our backlog to tackle later?

Reply:
Oh I didn't check on it, good catch! Fixed in the last push